python_scripts/01_tabular_data_exploration_ex_01.py

```diff
@@ -0,0 +1,62 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.6.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # 📝 Exercise 01
+
+# %% [markdown]
+# We will try to predict penguins species based on two of their body
+# measurements: culmen length and culmen depth.
```

Comment:
This sentence is weird because this is not what we actually do in the exercise here. Maybe this is a matter of phrasing?

"Assume we are interested in predictive penguins species ..."